src/flow-manager.c

```diff
@@ -875,7 +877,10 @@ static TmEcode FlowManager(ThreadVars *th_v, void *thread_data)
         }
         if (other_last_sec == 0 || other_last_sec < (uint32_t)SCTIME_SECS(ts)) {
             if (ftd->instance == 0) {
-                DefragTimeoutHash(ts);
+                uint32_t defrag_cnt = DefragTimeoutHash(ts);
```

Comment:
fn should return `u16` or have a debug validate if that's exceeded? Or counter should be `u32`?

Reply:
ah nvm this one. Misunderstood by the name of the var.